mypy/nodes.py

```diff
@@ -254,12 +254,49 @@ def accept(self, visitor: NodeVisitor[T]) -> T:
         return visitor.visit_overloaded_func_def(self)
 
 
-class FuncItem(FuncBase):
-    args = None  # type: List[Var]  # Argument names
-    arg_kinds = None  # type: List[int]  # Kinds of arguments (ARG_*)
+class Argument(Node):
+    """A single argument in a FuncItem.
+    """
+    def __init__(self, variable: 'Var', type_annotation: 'Optional[mypy.types.Type]',
+            initializer: Optional[Node], kind: int,
+            initialization_statement: Optional['AssignmentStmt'] = None) -> None:
```

Comment:
Can we just always create the assignment statement in `__init__` instead of taking it as an argument, for consistency?